### PR TITLE
[RW-9208][risk=no] Remove the bypassable column on access_module

### PR DESCRIPTION
**Pull request by Peter-Lavigne**
# Description

Remove the bypassable column on access_module and any code made redundant by this change.

This column was recently made obsolete in [this PR](https://github.com/all-of-us/workbench/pull/7543). See that PR for advice on testing locally.

This isn't the last PR for this story as I have at least one more refactor in mind.

---
**PR checklist**

- [x] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [x] This PR includes appropriate unit tests
- [x] I have added explanatory comments where the logic is not obvious
- [x] I have run and tested this change locally, and my testing process is described here
- [x] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [x] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [x] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [x] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [x] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/access/AccessModuleService.java [hunk 1]
```diff
@@ -40,9 +40,6 @@ Optional<AccessModuleStatus> getAccessModuleStatus(
    */
   boolean isModuleCompliant(DbUser dbUser, DbAccessModuleName accessModuleName);
 
-  /** Returns true if the access module is bypassable and bypassed */
-  boolean isModuleBypassed(DbUser dbUser, DbAccessModuleName accessModuleName);
```
Comment (Peter-Lavigne) [inline api/src/main/java/org/pmiops/workbench/access/AccessModuleService.java:44]: This function was unused.

File: api/src/test/java/org/pmiops/workbench/access/UserServiceAccessTest.java [hunk 2]
```diff
@@ -468,11 +468,10 @@ public void test_maybeSendAccessTierExpirationEmails_bypassed_is_up_to_date() {
         dbUser.getUserId(), DbAccessModuleName.DATA_USER_CODE_OF_CONDUCT, true);
     accessModuleService.updateBypassTime(
         dbUser.getUserId(), DbAccessModuleName.RT_COMPLIANCE_TRAINING, true);
-
-    // these 2 are not bypassable
```
Comment (Peter-Lavigne) [inline api/src/test/java/org/pmiops/workbench/access/UserServiceAccessTest.java:472]: I missed this test in the original PR, but could have updated it there.

File: ui/src/app/pages/access/data-access-requirements.spec.tsx [hunk 3]
```diff
@@ -235,12 +235,7 @@ describe('DataAccessRequirements', () => {
     const newModules = fp.map(
       (moduleStatus) => ({
         ...moduleStatus,
-        bypassEpochMillis:
-          // profile and publication are not bypassable.
-          moduleStatus.moduleName === AccessModule.PROFILECONFIRMATION ||
-          moduleStatus.moduleName === AccessModule.PUBLICATIONCONFIRMATION
-            ? null
-            : bypassFn(),
```
Comment (Peter-Lavigne) [inline ui/src/app/pages/access/data-access-requirements.spec.tsx:243]: Like the above, I missed this test file in the original PR but could have updated it there.

**Review by freemabd — APPROVED** [review]
Not super familiar will the access module, but removing a column seems easy enough. LGTM

**Comment by Peter-Lavigne** [comment]
I'm getting a completely different set of a few e2e test failures on a rerun. They don't appear to be related to the changes. As a result, I think they are false negatives and am merging this as-is.

File: api/db/changelog/db.changelog-211-remove-bypassable-from-access-module.xml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet author="Peter-Lavigne" id="db.changelog-211-remove-bypassable-from-access-module">
+        <preConditions onFail="MARK_RAN" >
```
Comment (jmthibault79) [inline api/db/changelog/db.changelog-211-remove-bypassable-from-access-module.xml:8]: ooh I'll have to remember this one

**Pull request merged**